### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by darrenmothersele** [kendraio/kendraio-app#34]
WIP documentation is availble on Google Docs here:
https://docs.google.com/document/d/1eriDR_5bCBAhNMdFD4SNc6erQI1rsE2j0idNyMDBX4I/edit#

**Comment by darrenmothersele** [comment]
Would like to move documentation into this repo, alongside the project.

Blocked by https://github.com/kendraio/kendraio-app/issues/13
as /docs folder currently used to host builds.

**Comment by darrenmothersele** [comment]
Now that #13 is done, I am going to reclaim the docs folder for documentation. To start I will move over the workflow/blocks documentation from the Google Doc.

We will be hosting the `/docs` folder on [Read the Docs](https://readthedocs.org/).

Please first read [An introduction to Sphinx and Read the Docs](http://www.ericholscher.com/blog/2016/jul/1/sphinx-and-rtd-for-writers/) for help contributing to the docs.

**Comment by darrenmothersele** [comment]
First version is up and running on Read the Docs:

https://kendraio-app.readthedocs.io

**Comment by lukestanley** [comment]
Yes, it works. People can make a new ticket for any bugs. @gsambrotta
